packages/main/src/TabContainer.js

```diff
@@ -912,6 +979,9 @@ class TabContainer extends UI5Element {
 			if (tab.getTabInStripDomRef() && !tab.getTabInStripDomRef().hasAttribute("hidden")) {
 				focusableTabs.push(tab);
 			}
+			if (tab.requiresExpandButton) {
+				focusableTabs.push(tab.getDomRef().querySelector("[ui5-button]"));
+			}
```

Comment:
Don't include this in the tab chain. The 'expand button popover ' should only be shown upon pressing 'arrow key down'.